analyses/country_template/docs/template.Rmd

```diff
@@ -0,0 +1,51 @@
+---
+title: '<img src="https://drive.google.com/uc?id=118y5T73-iSUZoAMtoJGddxq9QzD_GDKX" style="height:40px;float:left; margin:10px" /><img src="https://drive.google.com/uc?id=1fHQUzF3ZjaoHj9KQ33-94dK_X1hcmjzW" style="height:50px;float:left; margin:10px" />'
+pagetitle: 'Docs Template'
+output:
+  html_document:
+    df_print: paged
+    toc: yes
+    toc_float: yes
+    toc_depth: 4
+  pdf_document:
+    toc: yes
+    toc_depth: '4'
+editor_options:
+  chunk_output_type: inline
+---
+
+<br> 
+<br>
+<br>
+
+# Documentation Template
+Analysis last updated `r Sys.Date()`
+
+<br>
+
+```{r setup, include = FALSE, message = FALSE, warning = FALSE}
+# source("") # Link to external script
+library(kableExtra)
+library(knitr)
+knitr::opts_chunk$set(echo = FALSE) # do not print code by default
+knitr::opts_chunk$set(include = FALSE) # do not print output by default
+```
+
+
+Copy this basic template to get started with creating an R Markdown file to document an analysis.
+
```

Comment:
Do we want to provide a TOC/section template? Like Background, etc.? I don't see it as necessary but if we bother creating a template, would be worth providing some structure so similar info is documented between analyses.

Reply:
I'd say that we probably don't yet have enough consistency in what goes into these docs to make that kind of template meaningful yet. Hopefully this can develop further as we use this kind of documentation more